web/src/features/moreCast2/util.test.ts

```diff
@@ -11,15 +11,22 @@ import {
   rowIDHasher,
   validActualPredicate,
   validForecastPredicate,
-  fillForecastsFromRows
+  fillForecastsFromRows,
+  dateTimeComparator,
+  calculateFWIs,
+  simulateFireWeatherIndices
 } from 'features/moreCast2/util'
 import { buildValidActualRow, buildValidForecastRow } from 'features/moreCast2/rowFilters.test'
 import { createEmptyMoreCast2Row } from 'features/moreCast2/slices/dataSlice'
+import { MoreCast2Row } from 'features/moreCast2/interfaces'
 
 const TEST_DATE = '2023-02-16T20:00:00+00:00'
 const TEST_DATE2 = '2023-02-17T20:00:00+00:00'
 const TEST_CODE = 209
 const TEST_DATETIME = DateTime.fromISO(TEST_DATE)
+const YESTERDAY = DateTime.now().plus({ days: -1 })
```

Comment:
Might want to hard code these to a specific date for repeatability but could see having new dates tested every day as valuable.